x/pool/keeper/msg_server_update_pool_test.go

```diff
@@ -322,33 +322,12 @@ var _ = Describe("msg_server_update_pool.go", Ordered, func() {
 		Expect(pool.Name).To(BeEmpty())
 	})
 
-	It("Update pool with invalid UploadInterval", func() {
-		// ARRANGE
-		msg := &types.MsgUpdatePool{
-			Authority: gov,
-			Id:        1,
-			Payload:   "{\"UploadInterval\": 0}",
-		}
-
-		p, _ := BuildGovernanceTxs(s, []sdk.Msg{msg})
-
-		// ACT
-		_ = s.RunTxError(&p)
-		s.Commit()
-
-		// ASSERT
-		pool, found := s.App().PoolKeeper.GetPool(s.Ctx(), 0)
-
-		Expect(found).To(BeTrue())
-		Expect(pool.Name).To(BeEmpty())
-	})
```

Comment:
@troykessler This should not be removed.

Reply:
It is a duplicate testcase, that's why I removed it (compare L304 and L325 on main)